### ISSUE TITLE
bug: fullnameOverride is modified by adding a prefix 

### ISSUE THREAD

**Issue by 1mr** [VictoriaMetrics/helm-charts#1628]
**Chart name and version**
chart: victoria-metrics-cluster
version: v0.14.6

**Describe the bug**
Using the `fullnameOverride` key, the pod/svc/deployment/... name is expected to be the value of this key, but a prefix is always added. v0.13.0 was used before w/o this bug.

**Custom values**
Please provide only custom values (excluding default ones):
```yaml
vmselect:
  fullnameOverride: 'vmselect-bla-bla'
  ....
vminsert:
  fullnameOverride: 'vminsert-bla-bla'
  ....
vmstorage:
  fullnameOverride: 'vmstorage-bla-bla'
  ....
```

**Pod/svc name**
```
kubectl get pod
NAME                                         READY   STATUS    RESTARTS   AGE
vmagent-0                                    1/1     Running   0          3h33m
vmagent-1                                    1/1     Running   0          3h33m
vmagent-bla-bla-0                      1/1     Running   0          3h33m
vminsert-bla-bla-vminsert-5976f7fd-p52jm     1/1     Running   0          19s
vminsert-bla-bla-vminsert-5976f7fd-psggz     1/1     Running   0          19s
vmselect-bla-bla-vmselect-696cd8d894-8n5xd   1/1     Running   0          19s
vmselect-bla-bla-vmselect-696cd8d894-nvm5b   1/1     Running   0          19s
vmstorage-bla-bla-vmstorage-0                0/1     Pending   0          19s
```

**Comment by AndrewChubatiuk** [comment]
thanks for submitting an issue! fix will be available in the next release

**Comment by AndrewChubatiuk** [comment]
fix should be available in 0.14.7
